src/mainwindow.cpp

```diff
@@ -532,14 +532,22 @@ void MainWindow::setupKeyboardShortcuts()
     new QShortcut(QKeySequence(Qt::CTRL | Qt::Key_D), this, SLOT(deleteSelectedNote()));
     new QShortcut(QKeySequence(Qt::CTRL | Qt::Key_F), m_searchEdit, SLOT(setFocus()));
     new QShortcut(QKeySequence(Qt::CTRL | Qt::Key_E), m_searchEdit, SLOT(clear()));
+    
+    // Add macOS-specific shortcuts for note navigation
```

Comment:
```suggestion
    // shortcuts for note navigation
```
These are not just macOS specific ;)